fem/complex_fem.hpp

```diff
@@ -464,6 +474,11 @@ class ParComplexLinearForm : public Vector
    void AddDomainIntegrator(LinearFormIntegrator *lfi_real,
                             LinearFormIntegrator *lfi_imag);
 
+   /// Adds new Domain Integrator.
```

Comment:
Minor point but this comment should also mention the restriction to certain attributes